src/Octokit.Webhooks/Extensions/StringEnum.cs

```diff
@@ -7,7 +7,7 @@ namespace Octokit.Webhooks.Extensions;
 using JetBrains.Annotations;
 
 [PublicAPI]
-public sealed record StringEnum<TEnum>
+public sealed class StringEnum<TEnum> : IEquatable<StringEnum<TEnum>>
```

Comment:
Is this deliberately left a class instead of a struct or a readonly struct?

Reply:
Good idea! I hadn't considered that.